app/app.go

```diff
@@ -572,6 +591,38 @@ func (app *App) RegisterTendermintService(clientCtx client.Context) {
 	tmservice.RegisterTendermintService(app.BaseApp.GRPCQueryRouter(), clientCtx, app.interfaceRegistry)
 }
 
+// RegisterUpgradeHandlers returns upgrade handlers
+func (app *App) RegisterUpgradeHandlers(cfg module.Configurator) {
+	app.UpgradeKeeper.SetUpgradeHandler(upgradeName, func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		return app.mm.RunMigrations(ctx, cfg, vm)
+	})
+
+	app.UpgradeKeeper.SetUpgradeHandler(upgradeName, func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		// force an update of validator min commission
+		validators := app.StakingKeeper.GetAllValidators(ctx)
+		// hard code this because we don't want
+		// a) a fork or
+		// b) immediate reaction with additional gov props
+		minCommissionRate := sdk.NewDecWithPrec(5, 2)
```

Comment:
perhaps we can move this 5 elsewhere as a constant

Reply:
Made this into a constant named `minComRate`